steps/monitoring-apps-sizes/1.0.0/step.yml

```diff
@@ -0,0 +1,112 @@
+title: Mobile apps sizes monitoring
+summary: |
+  Check and monitor mobile applications sizes
+description: |
+  Check mobile applications sizes and compare with configured values to be warned when apps sizes increased
+
+  ### Usage
+
+
+  To give to our step the informations about the expected values, you have:
+  - create a config file (which should be added to your project repository) and set `config_file_path`  to find them. You can find a config example file [here](#config-file-example)
+  - **OR** set these keys with expected values directly on Bitrise side
+    - android_apk_size
+    - ios_ipa_size
+    - alert_threshold
+
+  ### Troubleshooting
+
+  - Be aware, you have to launch this step after [apps-decompiler](https://github.com/imranMnts/bitrise-step-apps-decompiler) step
+website: https://github.com/odemolliens/bitrise-step-monitoring-apps-sizes
+source_code_url: https://github.com/odemolliens/bitrise-step-monitoring-apps-sizes
+support_url: https://github.com/odemolliens/bitrise-step-monitoring-apps-sizes/issues
+published_at: 2021-01-21T12:51:25.302459+01:00
+source:
+  git: https://github.com/odemolliens/bitrise-step-monitoring-apps-sizes.git
+  commit: defe3c4cb32f111ad5cfecdd3b0bf5b7aa2c5a36
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- android
+- react-native
+- flutter
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+is_requires_admin_user: false
+is_always_run: true
```

Comment:
Do you want the `is_always_run` field to be `true`? 
```
is_always_run: if true the step will be executed even if a previous step failed during the build. 
Default is false.
- if type_tags is 'notification' this needs to be true
```

Reply:
We have to keep it to true because when a quality step (eg. monitoring-apps-permissions step) find something, it generates an error so it will cancel all other quality checks which will launch after this one.

I switched `is_always_run`to false on `apps-decompiler` step and we are checking at the top of the script if we have or not the decompiled app to not try to do something without that